README.md

```diff
@@ -1,3 +1,157 @@
 # redux-entitize
 
-Automated [redux](https://github.com/reactjs/redux)-state management for all your entities.
\ No newline at end of file
+Automated [redux](https://github.com/reactjs/redux)-state management for all your entities.
+
+`redux-entitize` maintains normalized entities in the state and provides:
+
+- an entities reducer
```

Comment:
...ships with:
- a **reducer**
- a set of **action creators** to populate your state with entity data
- a set of **selectors** to retrieve entities from the state